packages/core/README.md

```diff
@@ -351,6 +351,33 @@ Additionally to the regular fields, `primaryContrast` and `secondaryContrast` ar
 }
 ```
 
+#### dark theme
+
+To enable an additional dark theme, simply add `dark` under the `light` theme as follows below:
+
+```js
+ theme: {
+    themes: {
+      light: {
+        primary: 'black',
+        primaryContrast: 'white',
+        secondary: '#c0ffee',
+        secondaryContrast: '#de1e7e',
+      },
+      dark: {
+        primary: 'white',
+        primaryContrast: 'black',
+        secondary: '#de1e7e',
+        secondaryContrast: '#c0ffee',
+      },
+    },
+  },
+```
```

Comment:
As you have added default values for `dark`, this section is misleading, as it would even work without such configuration in the client.

Simply add an example for the `themes.dark` part in the above example.

Reply:
🏓 @dopenguin